### ISSUE TITLE
Alert container is too high for static uNav alerts

### ISSUE THREAD

**Issue by esteinborn** [nys-its/universal-navigation#192]
On a desktop, the static uNav, when displaying an alert, is set to a height far higher than the content it is displaying, resulting in a lot of extra space between the uNav and the site content.

Seems to be the height of _certain_ tablets (second image) but the mobile version is the same height, but cuts off content as well (third image)

Site: http://dcjs.ny.gov 
# Height is too tall for content
Image:
![image](https://cloud.githubusercontent.com/assets/308007/5904110/483beec6-a554-11e4-83c8-f1a8be4ba64f.png)

# Height is just right for certain tablet sizes (not all)
![image](https://cloud.githubusercontent.com/assets/308007/5904139/7ac5d654-a554-11e4-8c8c-70bac173b412.png)

# Height is too high and content cut off for mobile:
![image](https://cloud.githubusercontent.com/assets/308007/5904168/9d5d06b0-a554-11e4-8d61-bf3e7986e328.png)

**Comment by Goldieashe** [comment]
Examples of height on the iframe (281px) pushing content down on desktop:
![image](https://cloud.githubusercontent.com/assets/2806467/5904279/6120c316-a555-11e4-9cb3-56d078848a7f.png)
![image](https://cloud.githubusercontent.com/assets/2806467/5904283/6751daae-a555-11e4-9d09-b7eadf5072cc.png)
![image](https://cloud.githubusercontent.com/assets/2806467/5904309/975989a4-a555-11e4-86db-a3dfbbd9dd23.png)

In looking at the pages by resizing the screen to table and mobile phone size, it seems the 281 is meant to allow all of the text to be visible on the tablet (see next screenshot) - but of course that needs to be adjusted for on desktop
![image](https://cloud.githubusercontent.com/assets/2806467/5904431/50e8a346-a556-11e4-8ddb-2e16d7909fd9.png)

**Comment by ValPerrotte** [comment]
Shortly after the alert was posted, we received a notification from Acquia that they detected a service interruption affecting the site's availability on the Acquia Cloud platform. They found 404's
We are seeing lots of 404's with URLS on the backend --
 4 - HTTP 404 Not Found URLs                                                                                                                                                                                                                                                                                 Total: 33/33
 Top 404 Not Found URLs sorted by hits - [time served] [protocol] [method]

  63 11.65%  796.85 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=110
  43 7.95%   646.06 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=138
  40 7.39%   608.09 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=111
  16 2.96%   967.74 KiB HTTP/1.0 GET /autodiscover/autodiscover.xml
  8  1.48%   101.21 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=165
  5  0.92%    63.23 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=113
  5  0.92%    63.27 KiB HTTP/1.0 GET /xd_nygov.html?en=1&gn=0&host=http://static-assets.ny.gov/load_global_menu/ajax?iframe=true&height=117

And it appears that static-assets.ny.gov is from Akamai
static-assets.ny.gov is an alias for static-assets.ny.gov.edgekey.net.
static-assets.ny.gov.edgekey.net is an alias for e3531.g.akamaiedge.net.
e3531.g.akamaiedge.net has address 172.231.252.142

**Comment by Goldieashe** [comment]
@ValPerrotte did the alert banner show on that site for you? We had a couple of static unav sites which showed those type of errors and didn't display correctly
![image](https://cloud.githubusercontent.com/assets/2806467/5904612/dbf8b45c-a557-11e4-8976-0486a23dcddc.png)

**Comment by ValPerrotte** [comment]
No we did not get the alert to show.  I am on the phone with Acquia right now trying to get it resolved.

**Comment by Goldieashe** [comment]
Interesting- we noted that fourth level domains with the static uNav threw the errors for us and the alert banner did not show. Although one fourth level domain with https and the static unav did work.

**Comment by ValPerrotte** [comment]
It is now working on DMV.  The file did not exist in the docroot folder on the prod server.

**Comment by bernster** [comment]
I don't recall the purpose but during testing wasn't there a third yellow banner? I wonder if the unav is reserving space for it?

**Comment by bmtKIA6** [comment]
@esteinborn is this still an issue? 

cc
@lukecharde 
@tjcrommie

**Comment by lukecharde** [comment]
I don't believe it occurred in any dev testing... or even current dev sites - only in production for the live alert.

**Comment by esteinborn** [comment]
It's not happening on the DEV site. I'd be okay with closing and opening a new ticket when it happens again. COnfirm @Goldieashe?

**Comment by lukecharde** [comment]
Since it occurred on quite a few sites, it may be good to do a scheduled off-hours quick prod test for this one before closing - since it has only occurred in production.  @galinskim @christinacioffe Thoughts?

**Comment by christinacioffe** [comment]
Sure,  Mel would know best on timing for when to test

**Comment by Goldieashe** [comment]
Hello - as @esteinborn mentioned, it's not happening on our dev instances - dev iframe height seems to be set to 196px (was 281px in screenshots I put above). Testing off hours to confirm sounds good. Variable alert heights might be an issue for some sites

**Comment by galinskim** [comment]
In the evening would be best - like after 7PM.

**Comment by christinacioffe** [comment]
Let us know when you guys test
